controllers/user.js

```diff
@@ -167,11 +174,18 @@ async function getDashboard(req, res) {
   const user = _requireSessionUser(req, res);
   const allBreaches = req.app.locals.breaches;
   const { verifiedEmails, unverifiedEmails } = await getAllEmailsAndBreaches(user, allBreaches);
+  let lastAddedEmail = null;
```

Comment:
To avoid `let`, change to:

```js
const lastAddedEmail = req.session.lastAddedEmail;
if (lastAddedEmail) {
  delete req.session.lastAddedEmail; // or req.session["lastAddedEmail"] = undefined;
}
```

and you can drop the whole other `if (req.session.lastAddedEmail)` block.

Reply:
For reasons most mystérieux, `delete req.session.lastAddedEmail` does not work for me locally.